src/utils/definition-helpers.ts

```diff
@@ -73,8 +93,11 @@ export function getAbstractParts(root: string, prefix: string, collection: strin
 export function getAbstractPartsWithTemplates(root: string, prefix: string, collection: string, name: string) {
   return [
     [root, prefix, collection, `${name}.js`],
```

Comment:
discovering logic like this is part of semi-deprecated workflow, for new features we should use path marcher and global registry 
https://github.com/lifeart/ember-language-server/blob/component-context-info-origin/src/utils/path-matcher.ts#L74

Reply:
i see. let me update it. thanks :)